app/move/controllers/create/base.js

```diff
@@ -42,6 +44,23 @@ class CreateBaseController extends FormWizardController {
     next()
   }
 
+  async checkMoveSupported(req, res, next) {
```

Comment:
I've found these kind of methods easier to read when written using an early return pattern but that's purely personal preference:

```javascript
async checkMoveSupported(req, res, next) {
  const person = req.getPerson()

  if (!person || !person.prison_number) {
    return next()
  }

  const category = await personService.getCategory(person.id)
  if (!category || category.move_supported) {
    return next()
  }

  return res.render('move/views/create/move-not-supported', {
    person,
    category,
  })
}
```

The person check could even be simplified to (ie, if the person doesn't exist, by default they can't have a prison number?):

```javascript
  if (!person?.prison_number) {
    return next()
  }
```